types/react-native/index.d.ts

```diff
@@ -2215,6 +2215,12 @@ export interface AccessibilityPropsIOS {
      */
     accessibilityViewIsModal?: boolean;
 
+    /**
+     * When accessibile is true, the system will invoke this function when the user performs the escape gesture (scrub with two fingers).
+     * @platform ios
+     */
+    onAccessibilityEscape?: () => void;
```

Comment:
Technically the Flow version (`mixed`) maps to `unknown`, but looking at [this usage example](https://github.com/facebook/react-native/blob/5ee8202b2634cfba3527a4da7f30d042fa5699a6/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm#L560) it does seem like no return type is actually expected.

It would be good to modify this type upstream too, are you up for that? /cc @TheSavior

Reply:
Yeah, I was wondering about that as well. I'll make a PR for the FlowType soon :)

Reply:
We have the return type as mixed for many of these callbacks. It indicates that we don’t care about what you return from these functions. Sometimes people want to pass an async function (implicit return of a promise), or some other function they already have. 

If we only allow void return types then they have to wrap their callback in another function that doesn’t return anything just to satisfy the checker. So by allowing the return type of mixed the type checker is okay with it and we indicate we don’t look at the return value.

Reply:
Ah I see, thanks; TS is different in that regard 👍

Reply:
Ah I see, thanks; TS is different in that regard 👍